packages/hardhat-plugin/src/index.ts

```diff
@@ -123,6 +126,13 @@ ignitionScope
         })
       );
 
+      const deploymentId = resolveDeploymentId(givenDeploymentId, chainId);
+
+      const deploymentDir =
+        hre.network.name === "hardhat"
+          ? undefined
+          : path.join(hre.config.paths.ignition, "deployments", deploymentId);
+
       if (chainId !== 31337) {
         const prompt = await Prompt({
```

Comment:
Not for this PR. I think we should look at removing the prompt dependency. We are doing enough low level console work to justify putting in the equivalent to let us drop a dep.